jobs/resource_job.go

```diff
@@ -268,16 +268,17 @@ type JobSettings struct {
 	GitSource *GitSource `json:"git_source,omitempty"`
 	// END Jobs + Repo integration preview
 
-	Schedule             *CronSchedule         `json:"schedule,omitempty"`
-	Continuous           *ContinuousConf       `json:"continuous,omitempty"`
-	Trigger              *Trigger              `json:"trigger,omitempty"`
-	MaxConcurrentRuns    int32                 `json:"max_concurrent_runs,omitempty"`
-	EmailNotifications   *EmailNotifications   `json:"email_notifications,omitempty" tf:"suppress_diff"`
-	WebhookNotifications *WebhookNotifications `json:"webhook_notifications,omitempty" tf:"suppress_diff"`
-	NotificationSettings *NotificationSettings `json:"notification_settings,omitempty"`
-	Tags                 map[string]string     `json:"tags,omitempty"`
-	Queue                *Queue                `json:"queue,omitempty"`
-	RunAs                *JobRunAs             `json:"run_as,omitempty"`
+	Schedule             *CronSchedule            `json:"schedule,omitempty"`
+	Continuous           *ContinuousConf          `json:"continuous,omitempty"`
+	Trigger              *Trigger                 `json:"trigger,omitempty"`
+	MaxConcurrentRuns    int32                    `json:"max_concurrent_runs,omitempty"`
+	EmailNotifications   *EmailNotifications      `json:"email_notifications,omitempty" tf:"suppress_diff"`
+	WebhookNotifications *WebhookNotifications    `json:"webhook_notifications,omitempty" tf:"suppress_diff"`
+	NotificationSettings *NotificationSettings    `json:"notification_settings,omitempty"`
+	Tags                 map[string]string        `json:"tags,omitempty"`
+	Queue                *Queue                   `json:"queue,omitempty"`
+	RunAs                *JobRunAs                `json:"run_as,omitempty"`
+	Parameters           []JobParameterDefinition `json:"parameters,omitempty"`
```

Comment:
This field, similar to `Tasks` needs an additional annotation: `f:"alias:job_parameter"`.

We want users to be able to define one or more job parameters with the `job_parameter` block. The documentation should also reflect that.